.coveragerc

```diff
@@ -3,14 +3,19 @@ source = homeassistant
 omit =
     homeassistant/__main__.py
     homeassistant/helpers/signal.py
-    homeassistant/helpers/typing.py
-    homeassistant/scripts/*.py
+    homeassistant/scripts/__init__.py
+    homeassistant/scripts/check_config.py
+    homeassistant/scripts/ensure_config.py
+    homeassistant/scripts/benchmark/__init__.py
+    homeassistant/scripts/macos/__init__.py
+    homeassistant/scripts/ensure_config.py
+    homeassistant/scripts/benchmark/__init__.py
+    homeassistant/scripts/macos/__init__.py
```

Comment:
`homeassistant/helpers/typing.py` is at 100.00%, I'm not sure why it was in coverage ignore.

`homeassistant/scripts/auth.py` is at 95.59%, I think it makes sense to remove from ignore list as it's a core file.
I think it's also better to avoid the wildcard for core files.